dotnet/src/Microsoft.Agents.AI.DurableTask/AgentEntity.cs

```diff
@@ -21,7 +21,17 @@ internal class AgentEntity(IServiceProvider services, CancellationToken cancella
         ? cancellationToken
         : services.GetService<IHostApplicationLifetime>()?.ApplicationStopping ?? CancellationToken.None;
 
-    public async Task<AgentRunResponse> RunAgentAsync(RunRequest request)
+    public Task<AgentRunResponse> RunAgentAsync(RunRequest request)
```

Comment:
Should we add `ObsoleteAttribute` for this method and suggest consumers to consider the new `Run`?

Reply:
Didn't realize AgentEntity is internal.